pages/docs/overview/faqs.mdx

```diff
@@ -0,0 +1,61 @@
+# General FAQ
+
+#### What are the benefits of using Tangle Network for asset transfers?
+
+Tangle Network employs connected shielded pool protocols and zero-knowledge proofs to ensure privacy and security during asset transfers. It enables users to privately move and transfer assets between blockchains while obfuscating transaction history.
+
+#### How does Webb Protocol ensure the privacy and security of its users?
+
+Webb Protocol employs cutting-edge privacy technologies, such as zero-knowledge proofs, distributed key generation, and secure relayer systems, to maintain the privacy and security of users' assets and information.
+
+#### Can you explain the role of the Anchor System and the Relayer System in the Webb Protocol?
+
+The Anchor System connects merkle trees on different blockchains, ensuring liveness and safety. The Relayer System acts as an oracle, data relayer, and protocol governance participant, relaying information for connected Anchors and enhancing the overall security of the system.
+
+#### How does the Tangle Network maintain user privacy during cross-chain transfers?
+
+The Tangle Network's Connected Shielded Pool Protocol powers the ability to privately move and transfer assets between blockchains. It leverages zero-knowledge proofs to obfuscate transaction history and ensure the privacy of your assets during cross-chain transfers.
+
```

Comment:
Again, the Tangle network is not necessarily what maintains user privacy. It is what provides secure information transfer across blockchains via MPC protocol.